common/include/Utilities/Dependencies.h

```diff
@@ -267,6 +267,7 @@ extern const wxChar *__fastcall pxExpandMsg(const wxChar *englishContent);
 extern const wxChar *__fastcall pxGetTranslation(const wxChar *message);
 extern bool pxIsEnglish(int id);
 
+extern wxString fromUTF8(const std::string& str);
 extern wxString fromUTF8(const char *src);
```

Comment:
With `std::string_view`, they can be coalesced to a single function as string views provide "free" constructors from both `std::string` and `const char*`. You lose the luxury of a guaranteed null terminator, but those functions don't necessarily need it anyway.

Reply:
I'd prefer keeping the functions separate to add a clear difference between C++ strings and C strings, as those can be wide char on windows and a real pain in the arse to debug with bad conversions